pkg/frontend/openshiftpods_get.go

```diff
@@ -0,0 +1,53 @@
+package frontend
```

Comment:
nit: This file is called `openshiftpods_get.go` yet you are truly listing pods in a given namespace rather than "getting" a single pod. Could you rename the files to be `openshiftpods_list.go` pattern?

Reply:
New top pods and top nodes functions do implement rate limiting.

Files have been renamed to more accurately reflect functionality.